koala-app/apps/api/src/app/media/dto/update-media.input.ts

```diff
@@ -0,0 +1,8 @@
+import { CreateMediaInput } from './create-media.input';
+import { InputType, Field, Int, PartialType } from '@nestjs/graphql';
+
+@InputType()
+export class UpdateMediaInput extends PartialType(CreateMediaInput) {
+  @Field(() => Int)
+  id: number;
```

Comment:
I would assume id is not meant to be updated but the other fields like in the create input definition

Reply:
Id is needed -> see session